Makefile

```diff
@@ -7,7 +7,10 @@ EXTENSIONS = \
 	VMBackup
 
 clean:
-	rm -f build/*.zip
+	rm -rf build
+
+init:
+	@mkdir -p build
```

Comment:
Why not just make this the first step in the "build" recipe? I don't see the need for a separate target.